tests/callbacks/test_rich_progress_bar.py

```diff
@@ -201,3 +201,24 @@ def test_rich_progress_bar_refresh_rate(progress_update, tmpdir, refresh_rate, e
     trainer.fit(model)
 
     assert progress_update.call_count == expected_call_count
+
+
+@RunIf(rich=True)
+def test_rich_progress_bar_stages(tmpdir):
```

Comment:
nothing is being tested here?

Reply:
We missed testing for sanity check steps, hence we didn't catch the bug.

Reply:
Here we have two tests for the sanity check stage for an example: https://github.com/PyTorchLightning/pytorch-lightning/blob/ff20af41fd363f9520f5e0083d7a49051353cd1c/tests/callbacks/test_tqdm_progress_bar.py#L166

Reply:
Test updated!